apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -417,13 +417,11 @@ export const EventSetupTab = (
               <div className="mr-1.5 h-3 w-3">
                 <Icon name="check" className="h-3 w-3" />
               </div>
-              <Trans i18nKey="event_type_requres_google_cal">
+              <Trans i18nKey="event_type_requires_google_calendar">
                 <p>
                   The “Add to calendar” for this event type needs to be a Google Calendar for Meet to work.
                   Change it{" "}
-                  <Link
-                    href={`${WEBAPP_URL}/event-types/${formMethods.getValues("id")}?tabName=advanced`}
-                    className="underline">
+                  <Link href={`${WEBAPP_URL}/apps/google-calendar`} className="underline">
```

Comment:
Why is this link changed?

Also, could you test with a dummy translation for any non-english language that confirms that it works.

Reply:
In our code, we check if the conferencing app is Google Meet and the `destinationCalendar?.integration !== "google_calendar"`. If it isn't, we show this message. Previously, the link redirected to the event advanced tab, but this is unrelated.